apps/remix-ide/src/lib/gist-handler.js

```diff
@@ -60,6 +60,8 @@ function GistHandler (_window) {
           if (!errorLoadingFile) {
             const provider = fileManager.getProvider('workspace')
             provider.lastLoadedGistId = gistId
+          } else {
+            modalDialogCustom.alert(errorLoadingFile.message || errorLoadingFile)
```

Comment:
Please add a title to the modal, looks kind of unfinished without.

Reply:
I can update those, but most of the custom alert doesn't set any title..

Reply:
I know. I'm putting everywhere now. Looks better